selfservice/hook/verification_test.go

```diff
@@ -87,10 +86,13 @@ func TestVerifier(t *testing.T) {
 
 			h := hook.NewVerifier(reg)
 			require.NoError(t, hf(h, i, originalFlow))
-			s, err := reg.GetActiveVerificationStrategy(ctx)
-			require.NoError(t, err)
-			expectedVerificationFlow, err := verification.NewPostHookFlow(conf, conf.SelfServiceFlowVerificationRequestLifespan(ctx), "", u, s, originalFlow)
-			require.NoError(t, err)
+			assert.Lenf(t, originalFlow.ContinueWith(), 2, "%#ßv", originalFlow.ContinueWith())
+			assertContinueWithAddresses(t, originalFlow.ContinueWith(), []string{"foo@ory.sh", "bar@ory.sh"})
+			vf := originalFlow.ContinueWith()[0]
+			assert.IsType(t, &flow.ContinueWithVerificationUI{}, vf)
+			fView := vf.(*flow.ContinueWithVerificationUI).Flow
+
+			expectedVerificationFlow, err := reg.VerificationFlowPersister().GetVerificationFlow(ctx, fView.ID)
```

Comment:
This test assertion changed - previously we synthetically created a new verification URL and compared that with the assertion from the DB. Now we fetch both verification flows from the DB and compare them. Is this still a useful test?

Reply:
Do you mean removing https://github.com/ory/kratos/pull/3144/files/8169ac2945655590e9210777cf26b7766da2c4a5#diff-24ee502b250d67d70d70ba02ad18f303654b9af76552128958425b1c9043cc18R97-R100? I think that would still be fine, indeed.